modules/upgrade.adoc

```diff
@@ -21,7 +21,7 @@ All Kubernetes objects and PVs in each {product-title} cluster are backed up as
 
 [NOTE]
 ====
-The upgrade starting time might be later than the scheduled time. It could be sometimes an hour or longer.
+During a scheduled upgrade policy, it might take an hour or more for the upgrade to start, regardless of whether it is an immediate upgrade. Also, depending on the configuration of your workload, the upgrade might take much longer.
```

Comment:
```suggestion
During a scheduled upgrade policy, an upgrade might take an hour or more to start, regardless of whether it is an immediate upgrade. 

Also, depending on your workload configuration, the upgrade might take longer.
```

Just writing this out to reduce passive voice.

Reply:
The note serves as a heads-up when creating a recurring or individual upgrade policy. As far as I can tell, it usually takes at least an hour for the upgrade to begin. There would still be a delay in starting the upgrade even if it were launched immediately.

It also depends on the configuration of the customer's system.